web/src/App.jsx

```diff
@@ -8,11 +8,6 @@ import routes from './config/routes';
 function App() {
   return (
     <>
```

Comment:
you can just do empty carrots like this??

Reply:
Ya it's the shortened syntax for <React.Fragment>

Pretty sweet, no more pointless div wrappers.